ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/validator/SubmissionValidatorSpec.scala

```diff
@@ -251,10 +264,13 @@ class SubmissionValidatorSpec extends AsyncWordSpec with Matchers with Inside {
         .thenReturn(Future.successful(99))
       val logEntryAndStateResult = (aLogEntry(), someStateUpdates)
       val instance = new SubmissionValidator(
-        new FakeStateAccess(mockStateOperations),
-        (_, _, _, _, _) => logEntryAndStateResult,
+        ledgerStateAccess = new FakeStateAccess(mockStateOperations),
+        processSubmission = (_, _, _, _, _) => logEntryAndStateResult,
```

Comment:
Not in this PR but this is an opportunity for a welcome readability improvement

Reply:
I totally agree.